src/styles/index.scss

```diff
@@ -1,3 +1,62 @@
 body {
   margin: 0;
 }
+
+.calendar {
+  position: absolute;
+  top: 50%;
+  left: 50%;
+  transform: translate(-50%, -50%);
+  display: flex;
+  flex-wrap: wrap;
+  gap: 1px;
+  width: 706px;
+  padding: 10px;
+
+  &__day {
+    position: relative;
+    width: 100px;
+    height: 100px;
+    box-sizing: border-box;
+    background-color: #eee;
+    border: 1px solid black;
+    cursor: pointer;
+    transition: 500ms;
+    display: flex;
+    align-items: center;
+    justify-content: center;
+
+    @for $i from 1 through 31 {
+      &:nth-child(#{$i})::before {
+        content: '#{$i}';
+        font-family: Arial, Helvetica, sans-serif;
+        font-size: 30px;
+      }
+    }
+
+    &:hover {
+      background-color: #ffbfcb;
+      transform: translateY(-16, 8px);
```

Comment:
The transform property value 'translateY(-16, 8px)' seems to be incorrect. It should be either 'translateY(-16px)' or 'translate(-16px, 8px)'.